web/css/wv.alert.css

```diff
@@ -65,6 +65,7 @@
   position: absolute;
   font-size: 12px;
   right: 10px;
+  top: 4px;
```

Comment:
It might be a good idea to add a z-index here; something like z-index: 33; only because when I looked at this the other day, if the icon was overlaying another element it was partially hidden unless the z-index was set. Just a bit of a safety rule to ensure the element always appears on top.